.github/workflows/ci.yml

```diff
@@ -23,6 +23,16 @@ env:
   TRAVIS_BUILD_DIR: ${{github.workspace}}
 
 jobs:
+  golangci:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Clone the repo
+        uses: actions/checkout@v2
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v1
+        with:
+          # Keep this in sync with the version in envinit make target.
```

Comment:
mention the same in the Makefile, to update here?

why not use docker here as well like with the rest, so then they are automatically in sync?

Reply:
this doesn't run on docker. it's a separate thing, what github calls "workflow action". it's just a parameter for whatever script is supposed to run from github.com/golangci/golangci-lint-action, like a command line argument.

I added this step to have nice annotations right in pull requests, but it always requires a version to be specified.

made a reminder note in the Makefile as well, to keep it in sync with this one.